content/documentation/frontend-layer/layouts.md

```diff
@@ -8,30 +8,31 @@ aliases:
 
 # Layouts
 
-<p>
-<em>This document only applies when using <a href="https://github.com/gobuffalo/buffalo/tree/master/render" rel="nofollow">https://github.com/gobuffalo/buffalo/render</a>.</em><br>
-<em>Please see <a href="https://github.com/gobuffalo/plush" target="_blank">github.com/gobuffalo/plush</a> for more details on the underlying templating package.</em>
-</p
+{{<note>}}
+This document only applies when using [https://github.com/gobuffalo/buffalo/tree/main/render](https://github.com/gobuffalo/buffalo/tree/main/render).
+Please see [github.com/gobuffalo/plush](https://github.com/gobuffalo/plush) for more details on the underlying templating package.
+{{</note>}}
 
 ## Using a Standard Layout
 
 It is quite common to want to use the same layout across most, if not all of an application. When creating a new `render.Engine` the `HTMLLayout` property can be set to a file that will automatically be used by the `render.HTML` function.
 
+{{<codetabs>}}
```

Comment:
Personally, I don't like the code tab since it makes me hard to see the related things all together in the same view just by rolling my eyes without clicking tabs one by one or back to the previous tab.

(also, a small issue for someone like me is that I usually read docs with a narrow browser on the left side of my laptop monitor along with my code editor on the right side, and the narrow window makes the tabs ugly. Not sure how many developers are reading docs like me but...)

![image](https://user-images.githubusercontent.com/69406/174277743-991eed39-b733-42b2-80bc-76f23ba35c0e.png)

Reply:
Yes. Agree @sio4. We might want to tweak the component in general to improve its UX.

Reply:
We should open a ticket for it.